src/lib/types.ts

```diff
@@ -286,13 +316,17 @@ export interface ApiMachineConfig {
 export interface ApiMachineGuest {
   cpu_kind?: string
   cpus?: number
-  gpus?: number
```

Comment:
Another breaking change. Do you know if `gpus` field is no longer accepted by fly api?

Reply:
I wonder if they replaced it with something like `--vm-gpu-kind`, I will research in the morning. I doubt they would drop support for this

One thing interesting was on the region list, they hard code the regions were gpus are available: 
https://github.com/superfly/flyctl/blob/2bee8e741c08d9a3e01c7414ee110eb80e5f77ba/internal/command/platform/regions.go#L22